src/models/searchservice/response/Source.ts

```diff
@@ -4,7 +4,7 @@
  * @public
  */
 export enum Source {
-  /** The result is from an Answers Knowledge Graph. */
+  /** The result is from a Search Knowledge Graph. */
```

Comment:
I'd probably have this just `The result is from Knowledge Graph`, "Answers Knowledge Graph" was a little confusing

Reply:
like it 👍